schemas/test/en/test_first_non_empty_item_cross_section_dependencies.json

```diff
@@ -0,0 +1,376 @@
+{
+    "mime_type": "application/json/ons/eq",
```

Comment:
And some descriptions/guidance on date range question since I don't know what range is allowed etc.